package.json

```diff
@@ -10,14 +10,20 @@
   ],
   "scripts": {
     "build": "lerna run build",
-    "clean": "lerna run clean && rimraf node_modules",
+    "build:browser": "yarn build:packages && yarn build:extensions && yarn symlink:browser && yarn theia:browser build",
```

Comment:
If we are already splitting the build into browser and electron, maybe we should do the same for watching, what do you think?
```
    "watch:browser": "lerna run --parallel --ignore=\"crossmodel-app\" watch",
    "watch:electron": "lerna run --parallel --ignore=\"crossmodel-browser-app\" watch"
```

Watching is very fast in general so it may not be necessary but adding those two scripts also doesn't hurt I think.

Reply:
Good point, added the specific watch scripts and updated the README.